eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/io/DbsnpSubmittedVariantWriterTest.java

```diff
@@ -138,8 +145,12 @@ public void failsOnDuplicateVariant() throws Exception {
         DbsnpSubmittedVariantEntity variant = new DbsnpSubmittedVariantEntity(
                 EXPECTED_ACCESSION, hashingFunction.apply(submittedVariant), submittedVariant, 1);
 
-        thrown.expect(RuntimeException.class);
-        dbsnpSubmittedVariantWriter.write(Arrays.asList(variant, variant));
+        thrown.expect(BulkOperationException.class);
+        try {
```

Comment:
try and finally not needed (same as last comment)

Reply:
I have tested it, and if I remove the try and finally, all the lines after the one that throws the exception are silently ignored and the assertions are not checked. 

Another alternative to this is to remove the 'thrown.expect' call and use a try catch block to check the exception is being thrown, but personally I like this one (thrown + finally)